build.sbt

```diff
@@ -21,4 +21,6 @@ lazy val intellijHaskell = (project in file(".")).
     unmanagedJars in Compile += baseDirectory.value / "idea"
   )
 
-ideaBuild in ThisBuild := "163.7743.44"
\ No newline at end of file
+ideaBuild in ThisBuild := "163.7743.44"
+
+javacOptions ++= Seq("-encoding", "UTF-8")
```

Comment:
Yeah, it's already set in my IntelliJ.
Actually, I was stuck by the error caused by the source encoding when `compile` on sbt console.
Do you know how to set it on the console?
Sorry, I'm new to sbt.